docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx

```diff
@@ -0,0 +1,176 @@
+
+Let's check how to use the component!
+
+## Usage
+
+If you're using the [`MessagesScreen`](./01-messages-screen.mdx), the `SelectedReactionsMenu` is already set up for you. To use the `SelectedReactionsMenu` in your custom screens, simply add it to your UI, like so:
+
+```kotlin
+// The rest of your UI
+if (selectedMessageState is SelectedMessageReactionsState) {
+    val selectedMessage = selectedMessageState.message
+    SelectedReactionsMenu(
+        modifier = Modifier.align(Alignment.BottomCenter),
+        // The currently logged-in user
+        currentUser = user,
+        // The message whose reactions you selected
+        message = selectedMessage,
+        onMessageAction = { action ->
+            // Handle message action
+        },
+        onDismiss = { 
+            // Handle dismiss
+        }
+    )
+}
+
+```
+
+As you can see, showing the menu is very simple. If the `selectedMessageState` is an instance of `SelectedMessageReactionsState`, you pass in the the currently logged in user, as well as the selected message. The reactions you show are taken from the [`ChatTheme`](../02-general-customization/01-chat-theme.mdx) component and everything else required to show the component is taken care of internally.
+
+The aforementioned code produces the following UI:
+
+![Default SelectedReactionsMenu component](../../assets/compose_default_selected_reactions_menu.png)
+
+The component shows reactions from the [`ChatTheme`](../02-general-customization/01-chat-theme.mdx) on the top, followed by a list of message reactions.
+
+To dismiss the component just tap outside of the dialog or press the system back button.
+
+## Handling Actions
+
+Here are the actions exposed by the `SelectedReactionsMenu` component:
+
+```kotlin
+@Composable
+fun SelectedReactionsMenu(
+    ..., // State
+    onMessageAction: (MessageAction) -> Unit,
+    onDismiss: () -> Unit = {},
+)
+```
+
+* `onMessageAction`: Handler for the user performing an action such as leaving a reaction.
+* `onDismiss`: Handler when the menu is dismissed by clicking outside of the dialog area or by pressing the system back button.
+
+To handle these actions, you can override them like so:
+
+```kotlin
+if (selectedMessageState is SelectedMessageReactionsState) {
+    SelectedReactionsMenu(
+        ..., // State
+        onMessageAction = { action ->
+            composerViewModel.onMessageAction(action)
+			listViewModel.onMessageAction(action)
+        },
+        onDismiss = {
+			listViewModel.removeOverlay()
+        }
+    )
+}
+```
+
+In the snippet above, you propagate the `action` to the `composerViewModel` and `listViewModel`, for them to store the latest action. This will update the UI accordingly.
+
+Alternatively, you call `listViewModel.removeOverlay()` to remove the overlay from the screen, in `onDismiss()`. It's important to note that `onMessageAction()` calls `removeOverlay()` internally, to hide the overlay.
```

Comment:
Not sure if the comma in "...to remove the overlay from the screen**,** in `onDismiss()`." is necessary.

Same for "It's important to note that `onMessageAction()` calls `removeOverlay()` internally**,** to hide the overlay."